.github/workflows/gradle.yml

```diff
@@ -626,175 +626,6 @@ jobs:
           github-token: ${{ needs.find_valid_pat.outputs.pat }}
           ec2-instance-id: ${{ needs.start-platform-new-scheduler-acceptance-runner.outputs.ec2-instance-id }}
 
-  ## Kube Acceptance Tests
```

Comment:
Changes from me to test the issue with the acceptance test

Reply:
finally removed!